README.md

```diff
@@ -173,9 +173,31 @@ If you are using an older version of RN (i.e. `MainApplication.java` does not co
 ...
 ```
 
+#### Permissions on Android M and greater
+
+Because permissions need need to be granted at time of use in Android M, use [`react-native-android-permissions`](https://github.com/lucasferreira/react-native-android-permissions) to request permissions if you support Android M or above.
```

Comment:
`Because permissions need need to be granted` -> `Because permissions need to be granted`